main_routers/system_router.py

```
@@ -966,6 +968,86 @@ def _is_similar_to_recent_proactive_chat(lanlan_name: str, message: str) -> tupl
+
+            try:
+                if websocket.client_state == websocket.client_state.CONNECTED:
+                    await websocket.send_json({
+                        "type": "gemini_response",
+                        "text": message_text,
+                        "isNewMessage": True,
+                        "turn_id": turn_id,
+                        "skip_assistant_effects": True,
+                    })
+                    await websocket.send_json({"type": "system", "data": "turn end"})
+            except Exception as ws_error:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)